### ISSUE TITLE
more-info dialog tooltip

### ISSUE THREAD

**Issue by simenopsahl** [Madelena/Metrology-for-Hass#51]
On the more-info dialog for an entity, the tooltip when hovering over the graph or state history is generated outside the frame and hidden. A scrollbar appears at the bottom of the more-info card

**Comment by Madelena** [comment]
What version of HA are you on? Can you provide a screenshot? Thank you. 😀

**Comment by simenopsahl** [comment]
![Screenshot 2023-01-08 012752](https://user-images.githubusercontent.com/2676016/211175260-aa8fa094-8587-41df-8e91-eeafccfd450b.jpg)
The cursor wasn't included in the screenshot, but I am hovering on the highest peak.
This is on chrome. In developer tools the tooltip appears outside the frame as
`<div class ="chartTooltip  center" style="top: 321.998px; left: 752.237px;>...</div>"`
but it disappears as soon as I try to inspect it further because the I move the cursor 😅

Home Assistant 2023.1.1
Supervisor 2022.12.1
Operating System 9.4
Frontend 20230104.0

**Comment by simenopsahl** [comment]
It works fine on the android companion app, and I usually make custom popups so it's not critical.😁

**Comment by zanix** [comment]
I am also on 2023.1 and I had to update my fix from #6

```yaml
  card-mod-more-info-yaml: &card-mod-more-info-yaml |
    $: |
      @supports (-webkit-backdrop-filter: none) or (backdrop-filter: none) {
        .mdc-dialog__surface {
          backdrop-filter: blur(16px);
          -webkit-backdrop-filter: blur(16px);
          background-color: rgba(var(--rgb-mdc-theme-surface), .5) !important;
        }
      }
      .mdc-dialog .mdc-dialog__container .mdc-dialog__surface,
      .mdc-dialog .mdc-dialog__content {
        overflow: inherit;
      }
      .mdc-dialog__surface {
        box-shadow: 0 0px 16px 4px rgba(var(--dialog-box-shadow), .15), 0 16px 64px 32px rgba(var(--dialog-box-shadow), .5) !important;
      }
```

The added section is:

```yaml
      .mdc-dialog .mdc-dialog__container .mdc-dialog__surface,
      .mdc-dialog .mdc-dialog__content {
        overflow: inherit;
      }
```

However, there is a bug with this fix. When the popup loads, the history graph loads before the window resizes so it looks a bit funny.
